### PR TITLE
[BI-99] myBlogs improvements

### PR DESCRIPTION
**Pull request by LAlex14**
- slightly restyled + integrate correctly Paths for avatar && getFullName function
- commented Styleguide route-link from navBar to look like the final version at the presentation )

### CODE REVIEW

File: src/components/BaseCard.vue [hunk 1]
```diff
@@ -17,36 +17,38 @@
...
-          </p>
+          </div>
         </div>
         <div class="flex items-center justify-between">
           <div class="flex">
-            <avatar
-              class="w-10 h-10"
-              :path="get(post, 'relationships.creator.attributes')"
-            />
+            <avatar class="w-10 h-10" :path="getPath" />
```
Comment (cristijora) [inline src/components/BaseCard.vue:29]:
```suggestion
            <avatar class="w-10 h-10" :path="postCreator" />
```

File: src/components/BaseCard.vue [hunk 2]
```diff
@@ -67,7 +69,7 @@ export default {
   components: { ReadingTime, BaseTag },
   props: {
     post: {
-      type: Object,
+      type: [Object, String],
       default: () => {},
```
Comment (cristijora) [inline src/components/BaseCard.vue:73]: The default value will be `undefined` in this case
We should either convert to 
```suggestion
      default: () => ({}),
```
or 
```suggestion
      default: () => { return {} },
```
If we want an empty object as a default value

File: src/components/BaseCard.vue [hunk 3]
```diff
@@ -102,7 +104,14 @@ export default {
       const userId = get(this.post, 'relationships.creator.id', null);
       return userId == this.userState.id || this.$route.path == '/myblogs';
     },
+    getPath() {
```
Comment (cristijora) [inline src/components/BaseCard.vue:107]:
```suggestion
    postCreator() {
```

**Review by cristijora — APPROVED** [review]

**Review by alexstefan-binarcode — APPROVED** [review]
Check if there are no conflicts then merge

**Pull request merged**